### DISCUSSION TITLE
Read messages manually in server

### DISCUSSION THREAD

**Discussion by asdiujvaosidujdposiadjapoij** [uNetworking/uWebSockets#1767] [Q&A] [answered]
All the examples I have seen use the `message` member to set up a callback for incoming websocket messages. Is there a way to read the messages manually, in a blocking way, like you would do with raw sockets? I have a python server using the `websockets` library, and (although it is not the usual, recommended way) I can do something like 

    ws.handshake(timeout=1)
    message = ws.recv()  # Message is composed from frames if necessary, I think 
    # Do something with the message
    ws.send("Some response")

Is there a way to do something analogous with uWebSockets?

**Comment by uNetworkingAB** [comment, marked as answer]
No
